packages/api-client/src/conversation/ConversationRole.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Wire
+ * Copyright (C) 2020 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+export enum DefaultConversationRole {
+  WIRE_ADMIN = 'wire_admin',
+  WIRE_MEMBER = 'wire_member',
+}
+
+export interface ConversationRole {
```

Comment:
```suggestion
export interface ConversationPermission {
```

Reply:
What's the resoning behind this?

Reply:
The conversation role is "wire_admin" or "wire_member". The conversation permission is the connection of conversation role and allowed actions.

Reply:
> The conversation permission is the connection of conversation role and allowed actions.

But everywhere else the endpoints and interfaces which include this interface are calling it `ConversationRole` (see https://staging-nginz-https.zinfra.io/swagger-ui/#!/conversations/getConversationsRoles). Should we maybe rename `DefaultConversationRole` to `DefaultConversationRoleName` to avoid confusion?

In general, I try to apply the naming from wire-server, in this case I got it from [Roles.hs#L82](https://github.com/wireapp/wire-server/blob/e7e57d809253a3c55e7f0b3a01034c46647b0574/libs/galley-types/src/Galley/Types/Conversations/Roles.hs#L82)

Reply:
Ok, let's use `DefaultConversationRoleName` because I find it confusing that a `ConversationRole` interface has a member called `conversation_role`.